botocore/session.py

```diff
@@ -214,6 +214,17 @@ def _create_csm_monitor(self):
             return handler
         return None
 
+    def _get_crt_version(self):
+        try:
+            import pkg_resources
+            return pkg_resources.get_distribution("awscrt").version
```

Comment:
Hmm, a little unfortunate we have to do this. We should try and get the version directly on the module and rip this out when we get the chance.

Reply:
Yep, I filed awslabs/aws-crt-python#257 to try to get us off of this syntax. It's our only option right now though unfortunately.